pkg/render/logstorage.go

```diff
@@ -67,14 +67,14 @@ const (
 	ElasticsearchOperatorUserSecret       = "tigera-ee-operator-elasticsearch-access"
 	ElasticsearchAdminUserSecret          = "tigera-secure-es-elastic-user"
 
-	KibanaHTTPSEndpoint    = "https://tigera-secure-kb-http.tigera-kibana.svc.%s:5601"
-	KibanaName             = "tigera-secure"
-	KibanaNamespace        = "tigera-kibana"
-	KibanaPublicCertSecret = "tigera-secure-kb-http-certs-public"
-	TigeraKibanaCertSecret = "tigera-secure-kibana-cert"
-	KibanaDefaultCertPath  = "/etc/ssl/kibana/ca.pem"
-	KibanaBasePath         = "tigera-kibana"
-	KibanaServiceName      = "tigera-secure-kb-http"
+	KibanaName               = "tigera-secure"
+	KibanaNamespace          = "tigera-kibana"
+	KibanaPublicCertSecret   = "tigera-secure-es-gateway-http-certs-public"
```

Comment:
I suppose we can simplify the secret name to `tigera-es-gateway`.

Reply:
I think all of them should have the `secure` in it.